src/Collection/Collection.router.ts

```diff
@@ -348,9 +350,11 @@ export class CollectionRouter extends Router {
       )
     }
 
-    if (await this.isCollectionPublished(id)) {
+    const service = new CollectionService()
```

Comment:
The collection service is stateless, what do you think about having it as an instance variable?

Reply:
I like it! will change